lib/velum/kubernetes.rb

```diff
@@ -22,7 +22,7 @@ def self.kubeconfig
                                           targets:     "roles:kube-master",
                                           target_type: "grain",
                                           arg:         "cat /etc/pki/minion.key"
-      host = Minion.master.applied.pluck(:fqdn).first
+      host = Pillar.value(pillar: :apiserver) || Minion.master.applied.pluck(:fqdn).first
```

Comment:
Yes, I will remove it, it was to be extra safe here, but it doesn't make sense if we require it already.